disco_aws_automation/disco_storage.py

```diff
@@ -195,16 +192,6 @@ def configure_storage(self,
         logging.debug("mapped %s to root partition", disk_names[current_disk])
         current_disk += 1
 
-        # Map the latest snapshot for this hostclass
-        if map_snapshot:
-            latest = self.get_latest_snapshot(hostclass)
-            if latest:
-                self.wait_for_snapshot(latest)
-                current_name = disk_names[current_disk]
-                bdm[current_name] = self.create_snapshot_bdm(latest, iops)
-                logging.debug("mapped %s to snapshot %s", current_name, latest.id)
-                current_disk += 1
-
         # Map extra disk
         if extra_disk:
```

Comment:
i maybe wrong on the xvdz part. i guess we should do some testing by launching different instance type and assign snapshot to be mount to xvdz first. if it all works. then the code should be ok